manim/scene/three_d_scene.py

```diff
@@ -71,20 +71,34 @@ def begin_ambient_camera_rotation(self, rate=0.02):
         rate : int or float, optional
             The rate at which the camera should rotate about the Z_AXIS.
             Negative rate means clockwise rotation.
+        about: (str)
+            one of 3 options: ["theta", "phi", "gamma"]. defaults to theta.
         """
         # TODO, use a ValueTracker for rate, so that it
         # can begin and end smoothly
-        self.renderer.camera.theta_tracker.add_updater(
-            lambda m, dt: m.increment_value(rate * dt)
-        )
-        self.add(self.renderer.camera.theta_tracker)
+        if about == "phi":
+            x = self.renderer.camera.phi_tracker
+        elif about == "gamme":
+            x = self.renderer.camera.gamma_tracker
+        else:
+            x = self.renderer.camera.theta_tracker
```

Comment:
**Issue :** Why not raising an Exeption in this case ?

Reply:
Well, for no good reason. You are right.